podcasts/Sharing/SharingModal.swift

```diff
@@ -101,6 +101,16 @@ enum SharingModal {
     }
 
     static func show(option: Option, from source: AnalyticsSource, in viewController: UIViewController) {
+
+        if FeatureFlag.disablePrivateFeedSharing.enabled {
+            if case let .podcast(podcast) = option {
+                guard !podcast.isPrivate else {
```

Comment:
Nit: in this case `if podcast.isPrivate` is more readable

Reply:
Kind of prefer the `guard` for the early returns but it is more concise here with the `if case let`.